definitions/features/redis.rb

```diff
@@ -0,0 +1,22 @@
+class Features::Redis < ForemanMaintain::Feature
+  SCL_PREFIX = 'rh-redis5'.freeze
```

Comment:
IMHO this should be
```suggestion
  SCL_NAME = 'rh-redis5'.freeze
  SCL_PREFIX = '#{SCL_NAME}-'.freeze
```

Reply:
Isn't it overly verbose? Also, there will have to be double quotes

Reply:
A smidge, but this suggestion is based on terminology found in the SCL world. SCLs have names, and then a prefix. The prefix tends by convention to be the name + a dash. You don't have to get that verbose, its just what we in the packaging world are used to.

Reply:
It'd make EL8/Debian support easier as you'd then just can set PREFIX to `''`

Reply:
Turned into methods to allow easier conditionals based on platform